src/main/java/com/_up/megastore/services/implementations/UserService.java

```diff
@@ -29,39 +28,40 @@ public class UserService implements IUserService {
   private final IUserRepository userRepository;
   private final ITokenRepository tokenRepository;
   private final PasswordEncoder passwordEncoder;
-
+  private final ITokenService tokenService;
   @Value("${frontend.url}")
   private String frontendURL;
 
   public UserService(IUserRepository userRepository, PasswordEncoder passwordEncoder,
-      IEmailService emailService, ITokenRepository tokenRepository) {
+      IEmailService emailService, ITokenRepository tokenRepository, ITokenService iTokenService) {
```

Comment:
Usemos los nombres apropiados
```suggestion
      IEmailService emailService, ITokenRepository tokenRepository, ITokenService tokenService) {
```